app/src/main/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerViewModel.kt

```diff
@@ -102,6 +103,13 @@ class QuestionPlayerViewModel @Inject constructor(
     }
   }
 
+  fun getRawUserAnswer(
```

Comment:
Please add a KDoc here, too (should be the same as the one used for explorations, I think.

Reply:
Updated